articles/support/matrix.md

```diff
@@ -0,0 +1,801 @@
+    <td>**</td>
+  </tr>
+</table>
+
+### Auth0 Lock and Authentication
+
+The following matrix lists the level of support Auth0 will provide with regards to the operating system you have chosen when you are:
+
+* Using [Lock](/libraries/lock);
+* Performing authentication transactions.
+
+#### Mobile Operating Systems
+
+<table class="table">
+<tr>
+  <td>Apple iOS 9 (or later)</td>
+  <td>***</td>
+</tr>
+<tr>
+  <td>Google Android KitKat (4.4) or later</td>
+  <td>***</td>
+</tr>
+</table>
+
+## Software Development Kits (SDKs) and Libraries
+
+### Definitions
+
+In this section covering SDKs and Libraries, we will use the following terms to indicate the varying levels of support Auth0 will provide.
+
+<table class="table">
+  <tr>
+    <th>Certified</th>
+    <td>Tested and fully supported</td>
+  </tr>
+  <tr>
+    <th>Features</th>
+    <td>May receive new features</td>
+  </tr>
+  <tr>
+    <th>Bug Fixes</th>
+    <td>May receive bug fixes</td>
+  </tr>
+<table>
+
+### Lock
+
+<table class="table">
+  <tr>
+    <th>Library</th>
+    <th>Certified</th>
+    <th>Features</th>
+    <th>Bug Fixes</th>
+  </tr>
+  <tr>
+    <td>Lock.js</td>
+    <td>v10</td>
+    <td>v10</td>
+    <td>v9</td>
+  </tr>
+  <tr>
+    <td>Lock.swift</td>
+    <td>v2</td>
+    <td>v2</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Lock.iOS-macOS</td>
+    <td></td>
+    <td></td>
+    <td>v1</td>
+  </tr>
+  <tr>
+    <td>Lock.Android</td>
+    <td>v2</td>
+    <td>v2</td>
+    <td>v1</td>
+  </tr>
+</table>
+
+
+### Software Development Kits (SDKs)
+
+<table class="table">
+  <tr>
+    <th>SDK</th>
+    <th>Certified</th>
+    <th>Features</th>
+    <th>Bug Fixes</th>
+  </tr>
+  <tr>
+    <td>auth0.js</td>
+    <td>v7, v8</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0.Android</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0.Swift</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0.net</td>
+    <td></td>
+    <td>v3, v4</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>Auth0-java</td>
+    <td></td>
+    <td>v1</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>node-auth0</td>
+    <td></td>
+    <td>v2</td>
+    <td></td>
+  </tr>
+</table>
+
+### Frameworks/Platform Integration SDKs
+
+<div class="alert alert-info">
+  Auth0 reserves the right to downgrade support for an SDK to Community-Supported at any time.
+</div>
+
+<table class="table">
+  <tr>
+    <th>SDK</th>
+    <th>Certified</th>
+    <th>Features</th>
+    <th>Community-Supported</th>
+  </tr>
+  <tr>
+    <td>angular-auth0</td>
+    <td>X</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>angular-lock</td>
+    <td>X</td>
+    <td></td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>auth0-servlet</td>
+    <td></td>
+    <td></td>
+    <td>v3</td>
+  </tr>
+  <tr>
+    <td>auth0-spring-mvc</td>
+    <td></td>
+    <td></td>
+    <td>v1</td>
+  </tr>
+  <tr>
+    <td>auth0-spring-security-mvc</td>
```

Comment:
Let's make this `Features` since we are going to support this one but in a generic library instead of a specific one, and some paying customers are really using it